topk-js/src/data/function_expr.rs

```diff
@@ -0,0 +1,118 @@
+use napi::bindgen_prelude::*;
+use napi_derive::napi;
+
+use super::value::{BinaryVector, Vector};
+
+#[napi]
+#[derive(Debug, Clone)]
+pub enum FunctionExpression {
+    KeywordScore,
+    VectorScore { field: String, query: VectorQuery },
+    SemanticSimilarity { field: String, query: String },
+}
+
+#[napi(namespace = "query")]
+pub fn semantic_similarity(field: String, query: String) -> FunctionExpression {
+    FunctionExpression::SemanticSimilarity { field, query }
+}
+
+#[napi]
```

Comment:
does this need `namespace=query`?

Reply:
this will have cascade effect since then we would need to put under query namepace:
`FlexibleExpr -> Stage -> Query -> CollectionClient(since it returns query)`